generators/app/templates/parcel/app.js

```diff
@@ -1,4 +1,11 @@
 import { Elm } from "../Main.elm";
+import * as ElmDebugger from "elm-debug-transformer";
```

Comment:
Took me a bit to figure out, but imported like this, it actually still seems to increase the bundle size in production. However, it's possible to use the commonjs style `const ElmDebugger = require("elm-debug-transformer")` inside the conditional and have it fully removed in the prod build.